appservice/src/main/java/nu/yona/server/messaging/rest/BuddyMessageEmbeddedUserDecorator.java

```diff
@@ -28,12 +28,7 @@ public void decorate(MessageResourceAssembler assembler, MessageDto message)
 
 	private UserResourceAssembler createResourceAssembler(MessageResourceAssembler assembler, UserDto user)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1172)